templates/instruc_template.forTasks.rst

```diff
@@ -0,0 +1,236 @@
+:orphan: true
+	 
+.. Based on: https://dmtn-030.lsst.io/v/DM-7096/index.html#task-topic-type, with learnings from the 4 sfp pages built in branch DM-8717
+
+########################
+Guidance for ExampleTask
+########################
+
+We give guidance here for developers on what to write into the
```

Comment:
I'd like to see some development on the language here.

> We give guidance here for developers on what to write into the [..]

I think phrases like this can be iterated upon to be friendlier on the ear.